server/util/auth.js

```diff
@@ -0,0 +1,59 @@
+import * as Axios from 'axios';
+
+const config = require('../../config/server');
+
+const { LIKECOIN_AUTH_URL, LIKECOIN_USER_STUB } = config;
+
+const axios = Axios.create({
+  baseURL: `${LIKECOIN_AUTH_URL}`,
+});
+
+export async function getUserChallenge(queryWallet) {
+  let data;
+  if (process.env.production) {
+    ({ data } = await axios.get(LIKECOIN_AUTH_URL, {
```

Comment:
should be `process.env.NODE_ENV === 'production'` and might also want to check if `LIKECOIN_AUTH_URL` is not falsy